packages/block-library/src/columns/utils.js

```diff
@@ -96,10 +99,18 @@ export function getRedistributedColumnWidths(
  *
  * @return {boolean} Whether columns have explicit widths.
  */
-export function hasExplicitColumnWidths( blocks ) {
-	return blocks.every( ( block ) =>
-		Number.isFinite( block.attributes.width )
-	);
+export function hasExplicitPercentColumnWidths( blocks ) {
+	return blocks.every( ( block ) => {
```

Comment:
I think you can simplify this a bit like:
```
const blockWidth = block.attributes.width;
return Number.isFinite(
	blockWidth.endsWith?.( '%' ) ? parseFloat( blockWidth ) : blockWidth
);
```

Reply:
Ooooh, much better, thanks!